features/step_definitions/sample_file_steps.rb

```diff
@@ -0,0 +1,219 @@
+Given(/^a demo directory with a smelly file$/) do
+  contents = <<-EOS
+class Dirty
```

Comment:
But the dependency is on our tests only,  isn't it? Also active_support has a very low footprint these days.